src/material/list/selection-list.ts

```diff
@@ -202,6 +210,10 @@ export class MatListOption extends _MatListOptionMixinBase
     });
   }
 
+  ngOnChanges() {
```

Comment:
Could we reuse the `ngOnInit` to run this logic?

Reply:
Done.